### PR TITLE
Add tag with Polarion test case to DW test

### PR DESCRIPTION
**Pull request by jiripetrlik**

### CODE REVIEW

**Review by aloganat — DISMISSED** [review]

**Review by bdattoma — DISMISSED** [review]

File: ods_ci/tests/Tests/650__distributed_workloads/test-run-dw-tests.robot [hunk 1]
```diff
@@ -14,7 +14,8 @@ ${DW_JUNIT_FILE}          %{WORKSPACE=.}/junit.xml
 *** Test Cases ***
 Run distributed workloads sanity tests
     [Documentation]   Run tests located in Distributed Workloads repo downstream
-    [Tags]  Sanity
+    [Tags]  ODS-2511
```
Comment (jgarciao) [inline ods_ci/tests/Tests/650__distributed_workloads/test-run-dw-tests.robot:17]: Can you also add TierX tag?

File: ods_ci/tests/Tests/650__distributed_workloads/test-run-dw-tests.robot [hunk 2]
```diff
@@ -14,7 +14,8 @@ ${DW_JUNIT_FILE}          %{WORKSPACE=.}/junit.xml
...
+    ...     Sanity
 
     Skip If    "%{OPERATOR_TYPE}" != "RHODS Operator V2"
```
Comment (jgarciao) [inline ods_ci/tests/Tests/650__distributed_workloads/test-run-dw-tests.robot:20]: Can you use the new keyword  `Skip If Component Is Not Enabled` introduced in #1010

**Review by jgarciao — CHANGES_REQUESTED** [review]

**Comment by jiripetrlik** [comment]
@jgarciao Thank you for the review. I've added Tier1 tag. Yes, I will use "Skip If Component Is Not Enabled" when it is available.

**Comment by jstourac** [comment]
> @jgarciao Thank you for the review. I've added Tier1 tag. Yes, I will use "Skip If Component Is Not Enabled" when it is available.

#1010 just got merged JFYI.

**Review by jgarciao — APPROVED** [review]

**Review by asanzgom — APPROVED** [review]

**Pull request merged**